net-analyzer/sslsplit/sslsplit-0.5.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Transparent SSL/TLS interception"
+HOMEPAGE="http://www.roe.ch/SSLsplit"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="test"
+
+if [ "${PV}" == "9999" ] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/droe/${PN}"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="https://github.com/droe/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+RDEPEND="
+	elibc_musl? ( sys-libs/fts-standalone )
+	dev-libs/libevent[ssl,threads]
+	dev-libs/openssl:*"
```

Comment:
`dev-libs/openssl:0=` (if you have tons of time to spare, can you maybe also test libressl? Just an optional request, but would be nice)

Reply:
I tried to test it with libressl, however I wasn't able to compile it withe an error: ‘SSL {aka struct ssl_st}’ has no member named ‘state’. I tried to find some quick solution, but unsuccessfully.